### ISSUE TITLE
Error: no symbol named "CONSTANT" in "GMAP"

### ISSUE THREAD

**Issue by yaktender** [slburson/fset#48]
Error while loading in quicklisp through ultralisp

```
* (ql:quickload :fset)
To load "fset":
  Load 1 ASDF system:
    fset
; Loading "fset"
..................................................
[package fset]
debugger invoked on a SB-KERNEL:SIMPLE-PACKAGE-ERROR in thread
#<THREAD tid=259 "main thread" RUNNING {7008370473}>:
  no symbol named "CONSTANT" in "GMAP"
```

**Comment by slburson** [comment]
Thanks for the report.  The problem here seems to have been that Ultralisp wasn't picking up my Misc-Extensions system, on which FSet depends, and which I have also changed recently.  I have just now added Misc-Extensions to Ultralisp, which I believe will fix the problem.

(You had to have been getting Misc-Extensions somehow; I guess you were getting it through Quicklisp, which no longer has an up-to-the-minute version.)

Closing this in the expectation that the problem is fixed, but do reopen if it isn't.
